tests/test_reader.py

```diff
@@ -212,7 +212,7 @@ def test_get_outline(src, outline_elements):
         pytest.param(
             "imagemagick-ASCII85Decode.pdf",
             ["Im0.png"],
-            marks=pytest.mark.xfail(reason="broken image extraction"),
+            # marks=pytest.mark.xfail(reason="broken image extraction"),
```

Comment:
How does this change relate to this PR?

Reply:
It is not related to this fix. 
this is an xpass that was present for a long time and it is just a clean up. I don't this it worth to isolate this change in a dedicated PR